lib/kernel-manager.js

```diff
@@ -152,13 +154,13 @@ class KernelManager {
     });
   }
 
-  _executeStartupCode(kernel: Kernel) {
+  _executeStartupCode(kernel: ZMQKernel) {
     const displayName = kernel.kernelSpec.display_name;
     let startupCode = Config.getJson("startupCode")[displayName];
     if (startupCode) {
       log("KernelManager: Executing startup code:", startupCode);
       startupCode = `${startupCode} \n`;
-      // Looks like kernel.execute hasn't been implemented. When it is, will probably want to add an onResults fn to this call
+      // execute in ZMQKernel still wants another parameter
       kernel.execute(startupCode);
```

Comment:
We should make the [callback](https://github.com/nteract/hydrogen/blob/master/lib/zmq-kernel.js#L196) optional.